src/script.js

```diff
@@ -280,6 +280,10 @@ document.addEventListener('keydown', (event) => {
 
   if (event.code === 'Tab') {
     textarea.value += '  ';
+    event.preventDefault();
+  }
+  if (event.code === 'AltLeft' || event.code === 'AltRight') {
+    event.preventDefault();
```

Comment:
hmm... , what do you think about one preventDefault ?

Reply:
Moved extra preventDefault() in one option